drogon_ctl/templates/model_cc.csp

```diff
@@ -1709,7 +1709,7 @@ void [[className]]::get{%alias%}(const DbClientPtr &clientPtr,
             auto &pivotTargetKey=relationship.pivotTable().targetKey();
 %>
 std::vector<std::pair<{%relationshipClassName%},{%pivotTableClassName%}>> [[className]]::get{%alias%}(const DbClientPtr &clientPtr) const {
-    const static std::string sql = "select * from {%name%},{%pivotTableName%} where {%pivotTableName%}.{%pivotOriginalKey%} = <%c++
+    static const std::string sql = "select * from {%name%},{%pivotTableName%} where {%pivotTableName%}.{%pivotOriginalKey%} = <%c++
```

Comment:
What is the difference between the 'const static' and 'static const'?

Reply:
This is for style consistency. Whichever works, but it would be better to use only one of the two options.